src/features/movies/MovieList/styled.js

```diff
@@ -43,3 +43,47 @@ export const ListTitle = styled.h1`
 export const StyledLink = styled(Link)`
   text-decoration: none;
 `;
+
+export const PageChanger = styled.div`
+  display: flex;
+  margin: 40px 0 103px;
+  justify-content: center;
+`;
+
+export const Button = styled.button`
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  padding: 8px 16px;
+  border-radius: 5px;
+  font-size: 14px;
+  line-height: 1.4;
+  background-color: ${({ theme }) => theme.colors.grey};
+  margin-right: 12px;
+  border: none;
+  cursor: pointer;
+
+  ${(props) =>
+    props.forward &&
+    css`
+      background-color: ${({ theme }) => theme.colors.lightBlue};
+    `}
+`;
```

Comment:
Tutaj proponował bym zrobić małą zamianę, bo masz lewe guziki ciągle szare na każdej stronie. Proponuję, bazowo nadać na `background-color: ${({ theme }) => theme.colors.lightBlue}`, nadać style na `&:disabled`  i włożyć tam `background-color: ${({ theme }) => theme.colors.grey}` oraz gdy nie chcesz pointera dodać `pointer-events: none;`

Reply:
ale lewe guziki mają być szare xD

Reply:
lewe guziki w projekcie na figmie sa na szaro ale to jest podane jako przyklad, ze taki ma byc kolor jesli aktywna jest klasa disabled(Krzysiek wspominal o tym we wprowadzeniu), bazowy background-color jest niebieski